static/js/message_edit_history.js

```diff
@@ -61,6 +63,13 @@ export function fetch_and_render_message_history(message) {
                     edited_messages: content_edit_history,
                 }),
             );
+            // Pass the history through rendered_markdown.js
+            // to update dynamic_elements in the content.
+            $("#message-history")
+                .find(".rendered_markdown")
+                .each(function () {
```

Comment:
`.find()` method returns the first element that satisfies the given condition.
And then you are using the `.each()` method?
`.each()` method specifies a function to run for **each** matched element.

Reply:
This is [jquery `find()` method](https://api.jquery.com/find/) which returns all the children matching the selector.
I guess you were talking about [find method on `javascript` arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find).

Reply:
Oh! Okay, Cool!